app/src/main/res/values/strings.xml

```diff
@@ -928,4 +928,6 @@
     <string name="new_device_dialog_other_user_title">Your account “%1$s (@%2$s)” was used on:</string>
     <string name="new_device_dialog_other_user_message">%1$s\nFrom: %2$s \n\nIf you didn’t do this, remove the device (navigate to “Manage Devices” in the Settings section of this account), and reset your password.</string>
     <string name="new_device_dialog_other_user_btn">Switch Account</string>
+    <string name="label_system_message_joined_the_conversation_by_self">%1$s joined the conversation</string>
+    <string name="label_system_message_joined_the_conversation_by_other">%1$s joined the conversation</string>
```

Comment:
I think we already have this string in our strings file, `last_message_joined_conversation` can't we use it ?

Reply:
I did not notice that I will remove the new strings and use the old one

Reply:
i just double checked and they are not the same strings so we have to use those